arviz/data/io_pymc3.py

```diff
@@ -193,7 +202,7 @@ def sample_stats_to_xarray(self):
     @requires("model")
     def log_likelihood_to_xarray(self):
         """Extract log likelihood and log_p data from PyMC3 trace."""
-        if self.predictions:
+        if self.predictions or not self.log_likelihood:
             return None
         data = self._extract_log_likelihood()
```

Comment:
Extra idea. We could put a

    try:
         data = self._extract...
     except TypeError:
         warnings.warn("could not compute log likelihood. log_likelihood group will be omitted. Check your model object or set log_likelihood=False")
        return None

Which if I am correct should fix several issues such as https://github.com/arviz-devs/arviz/issues/395 (it has a minimal example to reproduce so it should be easy to check the issue is fixed) or even https://github.com/pymc-devs/pymc3/issues/3728

Reply:
Yes it does fix both the issues!